words/words.md

```diff
@@ -562,15 +562,15 @@ Nicht zu erwähnen die *weiteren* Maßnahmen, die wir ergreifen könnten, um R w
 
 . . .
 
-Wir hoffen, dass diese Möglichkeiten etwas Hoffnung schaffen. 
+Wir hoffen, dass diese Möglichkeiten etwas Hoffnung machen. 
 
-**Selbst bei einem pessimistischen Szenario *ist* es möglich COVID-19 zu bekämpfen und unsere Gesundheit und Wirtschaft zu schützen. ** Durch den *Lockdown* als "Reset", das gleichzeitige Drücken des Reproduktionsfaktors (R < 1), der Fallisolierung sowie einer breiten Verwendung einer datenschutzkonformen App zur Nachverfolgung der Kontakte und der Maskenpflicht... kann es gelingen, dass das Leben wieder in eine neue Normalität zurückkehrt.  
+**Selbst bei einem pessimistischen Szenario *ist* es möglich, COVID-19 zu bekämpfen und unsere Gesundheit und Wirtschaft zu schützen.** Durch den *Lockdown* als "Reset", das gleichzeitige Senken des Reproduktionsfaktors (R < 1) mit Fallisolierung sowie einer datenschutzkonformen App zur Nachverfolgung der Kontakte und *wenigstens* Alltagsmasken für alle ... kann es gelingen, dass das Leben wieder in sowas wie Normalität zurückkehrt.  
```

Comment:
hier m.E. besser: (...) in so etwas wie Normalität (...)

Reply:
:) Daran hänge ich mich nicht auf. Sehr gerne.